jmx-metrics/src/test/java/io/opentelemetry/contrib/jmxmetrics/InstrumenterHelperTest.java

```diff
@@ -446,6 +446,64 @@ void doubleValueCallbackMultipleMBeans() throws Exception {
                       .satisfiesExactlyInAnyOrder(assertDoublePoints()));
     }
 
+    @Test
+    void doubleValueCallbackMultipleAttributes() throws Exception {
```

Comment:
Could be a bit overkill, but would you mind adding a non-matching attribute name to this map or even a new case with all non-matching attributes (no valid values case)?

Reply:
Added nonmatching attribute